docs/Producing effective SARIF.md

```diff
@@ -609,4 +609,21 @@ Semantics: Assuming the reader of the log file (an end user or another tool) has
 
 ##### `Default`: note
 
-{0}: The file location '{1}' is specified with absolute URI. Prefer a relative reference together with a uriBaseId property.
\ No newline at end of file
+{0}: The file location '{1}' is specified with absolute URI. Prefer a relative reference together with a uriBaseId property.
+
+---
+### Rule `SARIF2025.ProvideRuleFriendlyName`
```

Comment:
>SARIF2025 [](start = 10, length = 9)

Just revert this file. I agree that it's not practical to keep it up to date manually.